src/Compatibility/ApiCompat/Microsoft.DotNet.ApiCompatibility/Rules/MembersMustExist.cs

```diff
@@ -152,7 +152,7 @@ private bool ParametersMatch(IMethodSymbol method, IMethodSymbol candidate)
 
             for (int i = 0; i < method.Parameters.Length; i++)
             {
-                if (!_settings.SymbolEqualityComparer.Equals(method.Parameters[i].Type, method.Parameters[i].Type))
```

Comment:
Can you please explain this change. I was really suprised when I saw that.

Reply:
I just noticed this when reviewing the code.  It looks to me like a copy-paste error when this change was introduced.  It would only be hit when searching a base type for a compatible method (as if a method was pushed down to a base type).  I think it would have excused method removals if the base type hierarchy contained an overload with the same number of parameters even if the type differed.